sklearn/ensemble/forest.py

```diff
@@ -41,7 +41,8 @@ class calls the ``fit`` method of each sub-estimator on random samples
 
 from __future__ import division
 
-import warnings
+from warnings import catch_warnings
```

Comment:
You can do multiple imports on the same line, ``from warnings import catch_warnings, simplefilter, warn``